src/DevYeah.LMS/DevYeah.LMS.Data/LMSContext.cs

```diff
@@ -37,7 +37,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
             if (!optionsBuilder.IsConfigured)
             {
 #warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
-                optionsBuilder.UseSqlServer("Server=DESKTOP-8A51TKG;Database=LMS;Trusted_Connection=True;");
```

Comment:
Do you know that even you changed the cold, the history will be there?

Fun fact: if you search "removed password", or "remove password" in all source code on GitHub, you will find lots of sensitive info.......